packages/driver/cypress/integration/commands/net_stubbing_spec.ts

```diff
@@ -1316,6 +1316,12 @@ describe('network stubbing', { retries: { runMode: 2, openMode: 0 } }, function
       .then(() => testDelay()).wait('@get')
     })
 
+    // @see https://github.com/cypress-io/cypress/issues/15901
+    it('can intercept utf-8 request bodies without crashing', function () {
+      cy.intercept('POST', 'http://localhost:5000/api/sample')
+      cy.visit('/fixtures/utf8-post.html')
```

Comment:
Since the `fetch(...)` is async you need to `cy.wait(...)` for the intercept to finish otherwise there's a race condition where the test could finish before the `fetch()` does and lead to a false negative.